### PR TITLE
move transcript dir out of plonk and into shared location

### PR DESCRIPTION
**Pull request by ledwards2225**
Move transcript dir from src/aztec/plonk/ to src/aztec/ since its contents will be used by both plonk and honk

### CODE REVIEW

File: cpp/src/aztec/plonk/proof_system/widgets/random_widgets/random_widget.hpp [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #pragma once
-#include "../../../transcript/transcript_wrappers.hpp"
+#include "../../../../transcript/transcript.hpp"
```
Comment (ledwards2225) [inline cpp/src/aztec/plonk/proof_system/widgets/random_widgets/random_widget.hpp:2]: This file only needed "transcript.hpp" which it was previously getting through "transcript_wrappers.hpp"

Comment (ledwards2225) [reply cpp/src/aztec/plonk/proof_system/widgets/random_widgets/random_widget.hpp:2]: I've made this change in 1-2 other files and also deleted several includes of transcript_wrappers.hpp wherever possible

Comment (codygunton) [reply cpp/src/aztec/plonk/proof_system/widgets/random_widgets/random_widget.hpp:2]: OK

File: cpp/src/aztec/honk/CMakeLists.txt [hunk 2]
```diff
@@ -1 +1 @@
-barretenberg_module(honk numeric ecc srs plonk proof_system)
+barretenberg_module(honk numeric ecc srs plonk proof_system transcript)
```
Comment (codygunton) [inline cpp/src/aztec/honk/CMakeLists.txt]: Why does honk depend on plonk?

Comment (codygunton) [reply cpp/src/aztec/honk/CMakeLists.txt]: I just testing removing it and had no issue. I guess you have made this possible with this PR or the proof_system move PR. Would you remove plonk as a dependency?

**Review by codygunton — CHANGES_REQUESTED** [review]
Looks good to me, just commented on one line.

**Review by codygunton — APPROVED** [review]

**Pull request merged**